client/lobby/SelectionTab.cpp

```diff
@@ -111,10 +111,32 @@ bool mapSorter::operator()(const std::shared_ptr<CMapInfo> aaa, const std::share
 	}
 }
 
+// pick sorting order based on selection
+static ESortBy SortBySelectionScreen(ESelectionScreen Type)
+{
+	// initialize 'generalSortingBy' early
+	switch(Type)
+	{
+	case ESelectionScreen::newGame:
+		return ESortBy::_name;
+	case ESelectionScreen::loadGame:
+	case ESelectionScreen::saveGame:
```

Comment:
consider using FALLTHROUGH macro (defined in Global.h) - the purpose of fallthrough is explained here: https://en.cppreference.com/w/cpp/language/attributes/fallthrough

Reply:
It is obvious. Even C# allows such constructs.